demo/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "demo",
+  "version": "1.0.0",
+  "main": "index.js",
```

Comment:
should we rename the `demo/index-es.js` ?

Reply:
removed package.main